### PR TITLE
Machine ID: Update documentation to reflect tbot CLI changes

### PR DESCRIPTION
**Pull request by timothyb89**
This updates the tbot documentation to reflect the new UX changes for v17, including:
* New `tbot start` and `tbot configure` subcommands 
* Runtime ACL management, with a note to avoid use of `tbot init`
* Bot instance management with `tctl bots instances`

### CODE REVIEW

File: docs/pages/reference/cli/tbot.mdx [hunk 1]
```diff
@@ -191,9 +200,101 @@ using database proxies.
...
+| Flag          | Description |
+|---------------|-------------|
+| `-o/--output` | If set, writes the generated configuration to the given file path instead of stdout. |
+
 ## tbot start
 
-Starts the Machine ID client `tbot`, fetching and writing certificates to disk at a set interval.
+The `tbot start` family of commands starts the Machine ID client in various
+modes, depending on the type of resources to be accessed. Refer to one of these
+for specific information:
```
Comment (ptgott) [inline docs/pages/reference/cli/tbot.mdx:253]:
```suggestion
modes, depending on the type of resources to be accessed:
```
So we don't include the unspecified "these"

**Review by ptgott — APPROVED** [review]

File: docs/pages/enroll-resources/machine-id/troubleshooting.mdx [hunk 2]
```diff
@@ -31,6 +31,13 @@ some additional context:
 
 ### Explanation
 
+<Notice type="note">
```
Comment (strideynet) [inline docs/pages/enroll-resources/machine-id/troubleshooting.mdx:34]: 👍  nice addition.

File: docs/pages/enroll-resources/machine-id/troubleshooting.mdx [hunk 3]
```diff
@@ -89,15 +96,15 @@ version: v2
 $ tctl rm lock/5cee949f-5203-4f3b-9805-dac35d798a16
 ```
 
-Next, reset the generation counter by deleting and recreating the bot:
+Next, use `tctl bots instances add` to generate a new join token for the bot:
 ```code
-$ tctl bots rm example
-
-$ tctl bots add example --roles=foo,bar
+$ tctl bots instances add example
```
Comment (strideynet) [inline docs/pages/enroll-resources/machine-id/troubleshooting.mdx:101]: Perhaps could be clearer that "example" here is the name of the bot ? Perhaps a short sentence before the instructions like "The following steps show rectifying this issue for a bot named "example""

**Review by strideynet — APPROVED** [review]

File: docs/pages/enroll-resources/machine-id/troubleshooting.mdx (hunk 3 above)
Comment (timothyb89) [reply docs/pages/enroll-resources/machine-id/troubleshooting.mdx:101]: Hmm, good point, we never called out the name of the bot. I've adjusted the text for the whole entry to make that clear in addition to repeating it here.

**Pull request merged**